lib/audit_help.c

```diff
@@ -48,7 +50,7 @@ void audit_help_open (void)
  * This function will log a message to the audit system using a predefined
  * message format. Parameter usage is as follows:
  *
- * type - type of message: AUDIT_USER_CHAUTHTOK for changing any account
+ * type - type of message: AUDIT_USER_MGMT for changing any account
  *	  attributes.
```

Comment:
Why do we only document this one type? Maybe we should actually point to some documentation that documents all of them, instead of documenting one and leaving the others undocumented?

Reply:
The documentation around the user and group account lifecycle wrt auditing is here: https://github.com/linux-audit/audit-documentation/wiki/SPEC-User-Account-Lifecycle-Events  This might help clarify some changes.

Reply:
I think we should point to that link, instead of arbitrarily documenting one of the types.

Reply:
I've added the link to the interfaces and to the commit message, so that it's easier to find